src/domain/tradingview/types.ts

```diff
@@ -8,3 +8,26 @@ export type Bar = BarType & {
 export type SymbolInfo = LibrarySymbolInfo & {
   isStable: boolean;
 };
+
+interface Flavoring<FlavorT> {
+  _type?: FlavorT;
+}
+type Flavor<T, FlavorT> = T & Flavoring<FlavorT>;
+
+/**
+ * This type ensures type safety when dealing with ordered arrays.
+ * @see {@link FromNewToOldArray}
+ */
+export type FromOldToNewArray<T> = Flavor<Omit<T[], "reverse">, "FromOldToNewArray"> & {
+  reverse(): FromNewToOldArray<T>;
+};
```

Comment:
maybe `AscOrderArray`/`DescOrderArray`?

from the first glance it looks like a helper type

Reply:
for me its just not that obviuos that asc = from old to new. I still have to spend a second or two to decode it to what it really means

Reply:
ok